open-api/rest-catalog-open-api.yaml

```diff
@@ -629,7 +887,7 @@ paths:
             The snapshots to return in the body of the metadata. Setting the value to `all` would
             return the full set of snapshots currently valid for the table. Setting the value to
             `refs` would load all snapshots referenced by branches or tags.
-          
+
```

Comment:
nit: remove unnecessary newline changes

Reply:
I noted this as well, but after thinking about it, I don't think the usual risk of merge conflicts really applies to this YAML file so I'm okay leaving this to keep it clean. I'm okay either way.